soapboxpb/environment.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package soapbox;
+option go_package = "soapboxpb";
+
+import "application.proto";
+
+service Environments {
+    rpc ListEnvironments (ListEnvironmentRequest) returns (ListEnvironmentResponse) {}
+    rpc GetEnvironment (GetEnvironmentRequest) returns (GetEnvironmentResponse) {}
+    rpc CreateEnvironment (CreateEnvironmentRequest) returns (CreateEnvironmentResponse) {}
+    rpc DestroyEnvironment (DestroyEnvironmentRequest) returns (DestroyEnvironmentResponse) {}
+    rpc CopyEnvironment (CopyEnvironmentRequest) returns (CopyEnvironmentResponse) {}
+}
+
+message ListEnvironmentRequest {}
+
+message EnvironmentSummary {
+    int32 id = 1;
+    string name = 2;
+}
+
+message ListEnvironmentResponse {
+    repeated EnvironmentSummary environments = 1;
+}
+
+message GetEnvironmentRequest {
+    int32 id = 1;
+}
+
+message GetEnvironmentResponse {
+    Environment env = 1;
+}
+
+message Environment {
+    int32 id = 1;
+    string name = 2;
+    string slug = 3;
+    bool productionFlag = 4;
```

Comment:
I think we can leave this off, because one of the design decisions is that the only thing that distinguishes a production environment is that it is the one taking traffic in the load balancer, so all environments can be treated equally from the POV of the data model.